include/aunit/framework/aunit-test_results.adb

```diff
@@ -60,7 +60,7 @@ package body AUnit.Test_Results is
    generic
       with function Test (Position : Result_Lists.Cursor) return Boolean;
    procedure Gen_Extract (R : in out Result;
-                          E : in out Result_Lists.List);
+                          E : out Result_Lists.List);
```

Comment:
I'm a bit confused. In the log you say that you are making the parameter in instead of out, but here it is now out instead of in.
I belive what you indended to to do was make the **first** parameter in instead ouf in out?

Reply:
Quite right, I intended to change the private result object, not the list.